vouches/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-encointer-vouches"
+version = "1.2.0"
+authors = ["encointer.org <alain@encointer.org> and Parity Technologies <admin@parity.io>"]
```

Comment:
Is parity technologies here with the outlook of being included in the fellowship repo?

Reply:
no, that's copy paste, thx